src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+    let partnerText;
+  
+    if (person.isMarried) {
+      partnerText = person.sex === 'f'
+        ? `${person.partnerName} is my husband`
+        : `${person.partnerName} is my wife`;
+    } else {
+      partnerText = 'I am not married';
+    }
```

Comment:
Try to simplify this logic to avoid repeating and `else` statements

Reply:
Firstly i tried to do this, but linter doesn't like nested ternary.